### DISCUSSION TITLE
Logs and the popup content at the top right corner

### DISCUSSION THREAD

**Discussion by Andras-Csanyi** [LazyVim/LazyVim#4851] [Q&A] [answered]
Hi,

I am trying to setup C# debugging and I have two issues, but none of them are related to C# and debugging, they are rather I don't know well LazyVim's ecosystem.

First one is the popup at the top-right corner. See the attached video.
How can I see those messages again? If I type `:messages` it doesn't show anything.

https://github.com/user-attachments/assets/f6ee2711-9bbf-425c-82b5-46e578576177

Second one is DAP logs. `:DapShowLog` shows something different. The displayed logs are from a month ago and not recent ones. Overall, is there a list of settings where I can check what directories are used by LazyVim?

**Comment by Kayzels** [comment, marked as answer]
To get everything that shows us there, you can use `<leader>sna`, which shows everything that Noice has previously displayed. I think it calls `:NoiceHistory` but not completely certain.

**Reply by Andras-Csanyi** [reply]
Thank you!
